libnmstate/nm/ipv4.py

```diff
@@ -96,6 +97,14 @@ def get_info(active_connection):
     if active_connection is None:
         return info
 
+    nm_ac_state = nmclient.NM.ActiveConnectionState
+    if active_connection.get_state() == nm_ac_state.ACTIVATING:
+        logging.debug(
+            'Interface {} is in activating state, showing saved config '
+            'in of running state '.format(active_connection.get_id())
```

Comment:
`in of` =>> `in the` ?

BTW, you can now use py3 cool stuff: `f'....{active_connection.get_id()}....'`